analyze_openapi_changes.py

```
@@ -0,0 +1,114 @@
+from typing import Dict, Any, Optional
+
+def load_json_file(file_path: str) -> Optional[Dict[str, Any]]:
+    try:
+        with open(file_path, 'r') as f:
+            return json.load(f)
+    except Exception as e:
+        print(f"Error loading {file_path}: {e}")
+        return None
+
+def analyze_changes(old_schema: Dict[str, Any], new_schema: Dict[str, Any]) -> Dict[str, Any]:
+    diff = DeepDiff(old_schema, new_schema, view='tree')
```

Ruff (0.8.2)

Local variable `diff` is assigned to but never used

Remove assignment to unused variable `diff`

(F841)